### DISCUSSION TITLE
Cross site scripting via Markdown

### DISCUSSION THREAD

**Discussion by Busata** [datenstrom/community#21] [Report a bug] [answered]
It's possible to add arbitrary javascript to the contents of the files. Not sure how to prevent it with PHP, could refer to http://blog.astrumfutura.com/2013/04/20-point-list-for-preventing-cross-site-scripting-in-php/.

You should look how to sanitize/escape your data with PHP

**Comment by markseuffert** [comment, marked as answer]
Security should be a first class citizen, that's why we added a safe mode.

Thanks again @Busata for reporting the bug, let me know if it comes back.
